### PR TITLE
Rename tests

### PR DESCRIPTION
**Pull request by mayzenator**
Wrote unit and benchmark test for rename/move. Currently the benchmark test has the actual performance metrics we want to measure commented out, and some smaller values in place for testing purposes. There seems to an issue with the timed performance logging because it's currently printing 0 ms for everything. Everything else is working and passing.

### CODE REVIEW

**Comment by ghost** [comment]
`feature/nn-performance` will be merged into develop with #85 . Please rebase/merge to develop.

**Comment by cannontwo** [comment]
I'm getting several lint errors right now that should be fixed: [lint_out.txt](https://github.com/comp413-2017/RDFS/files/1408706/lint_out.txt)

File: test/namenode/RenameTest.cc [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+TEST_F(NamenodeTest, movingPerformance) {
+    el::Loggers::setVerboseLevel(9);
+    std::vector<std::string> files;
+    hadoop::hdfs::CreateRequestProto create_req;
+    hadoop::hdfs::CreateResponseProto create_resp;
+    hadoop::hdfs::RenameRequestProto rename_req;
+    hadoop::hdfs::RenameResponseProto rename_resp;
+//    std::vector<int> depths = {5, 10, 100};
```
Comment (cannontwo) [inline test/namenode/RenameTest.cc]: I think you could uncomment these lines. Our test running script currently ignores all cases with `Performance` in the name, so it's okay to put the actual values here. We will only be running performance tests intentionally, not as part of Travis.

Comment (mayzenator) [reply test/namenode/RenameTest.cc]: fixed!

**Review by cannontwo — CHANGES_REQUESTED** [review]
Still some lint errors, then I'll approve

**Comment by cannontwo** [comment]
It looks like one of your tests is still failing

**Comment by ghost** [comment]
please stop committing from in the VM

**Review by LINKIWI — APPROVED** [review]
Approving to unblock; @eddiedugan is working on a patch to address the unit test failure in your build currently. It doesn't seem to be related to your change.

**Review by cannontwo — APPROVED** [review]

**Pull request merged**